planning/behavior_velocity_run_out_module/src/scene.cpp

```diff
@@ -102,9 +102,18 @@ bool RunOutModule::modifyPathVelocity(
   const auto dynamic_obstacles = dynamic_obstacle_creator_->createDynamicObstacles();
   debug_ptr_->setDebugValues(DebugValues::TYPE::NUM_OBSTACLES, dynamic_obstacles.size());
 
-  // extract obstacles using lanelet information
-  const auto partition_excluded_obstacles =
-    excludeObstaclesOutSideOfPartition(dynamic_obstacles, extended_smoothed_path, current_pose);
+  const auto filtered_obstacles = std::invoke([&]() {
+    // extract obstacles using lanelet information
+    const auto partition_excluded_obstacles =
+      excludeObstaclesOutSideOfPartition(dynamic_obstacles, extended_smoothed_path, current_pose);
+
+    // extract obstacles that cross the ego's back lane
+    if (!planner_param_.run_out.use_ego_cut_line) return partition_excluded_obstacles;
+
+    const auto ego_back_line_excluded_obstacles =
+      excludeObstaclesCrossingEgoBackLine(partition_excluded_obstacles, current_pose);
+    return ego_back_line_excluded_obstacles;
```

Comment:
```suggestion
    if (!planner_param_.run_out.use_ego_cut_line) return partition_excluded_obstacles;

    // extract obstacles that cross the ego's back lane
    const auto ego_back_line_excluded_obstacles =
      excludeObstaclesCrossingEgoBackLine(partition_excluded_obstacles, current_pose);
    return ego_back_line_excluded_obstacles;
```

Reply:
the comment place is better like this?

Reply:
Done, thanks for the suggestion! @kosuke55